docs/references/t-ecdsa-how-it-works.mdx

```diff
@@ -1,73 +1,114 @@
 
 Support for further elliptic curves and additional corresponding master keys may be added in the future. The curve `secp256r1` is interesting for supporting use cases such as decentralized certification authorities (CAs) and is the premier candidate group to be added to facilitate use cases like the mentioned one.
 
-## API
+## t-sig APIs
+
+Next, this guide gives an overview of the APIs for threshold signatures. 
+For the authoritative specification, the reader is referred to the corresponding part of the Internet Computer interface specification for [ECDSA](/references/ic-interface-spec.md#ic-ecdsa_public_key) and [Schnorr](/references/ic-interface-spec.md#ic-sign_with_schnorr). 
+The ECDSA API comprises two methods: `ecdsa_public_key` for retrieving threshold ECDSA public keys, and `sign_with_ecdsa` for requesting threshold ECDSA signatures to be computed from the subnet holding the secret-shared private threshold ECDSA key.
+The Schnorr API comprises two methods: `schnorr_public_key` for retrieving threshold Schnorr public keys, and `sign_with_schnorr` for requesting threshold Schnorr signatures to be computed from the subnet holding the secret-shared private threshold Schnorr key.
 
-Next, this guide gives an overview of the API for threshold ECDSA. For the authoritative specification, the reader is referred to the corresponding part of the [Internet Computer interface specification](/references/ic-interface-spec.md#ic-ecdsa_public_key). The API comprises two methods: `ecdsa_public_key` for retrieving threshold ECDSA public keys, and `sign_with_ecdsa` for requesting threshold ECDSA signatures to be computed from the subnet holding the secret-shared private threshold ECDSA key.
+Each API call refers to a t-sig master key by virtue of a 2-part identifier comprising a curve or algorithm, and a key id as outlined above. Derivation paths are used to refer to keys below a canister's root key in the key derivation hierarchy. The key derivation from the master key to the canister root key is implicit in the API.
 
-Each API call refers to a threshold ECDSA master key by virtue of a 2-part identifier comprising a curve and a key id as outlined above. Derivation paths are used to refer to keys below a canister's root key in the key derivation hierarchy. The key derivation from the master key to the canister root key is implicit in the API.
+-   [`ecdsa_public_key`](/docs/current/references/ic-interface-spec#ic-ecdsa_public_key): This method returns a SEC1-encoded ECDSA public key for the given canister using the given derivation path. 
+- If the `canister_id` is unspecified, it will default to the canister id of the caller. 
+- The `derivation_path` is a vector of variable length byte strings. 
+- The `key_id` is a struct specifying both a curve and a name. The availability of a particular `key_id` depends on implementation.
```

Comment:
Formatting is a bit weird in this part. I think we should have bullet points only for the API. The rest of the item referring to the same API should either be flattened (as in the original docs) or be indented more. Same for the other APIs

Reply:
@jessiemongeon1 do you have a suggestion here, or a normal style for API stuff? I think these actually look quite nice when rendered on the preview, but totally trust andrea.

Reply:
Ah, I see more what you mean, as everything is not bulleted, it looks like different lists. I'll propose something

Reply:
I think it's kind of better, but @jessiemongeon1 or @andreacerulli if you have suggestions, I'm open.

Reply:
LGTM!